include/Timeline.h

```diff
@@ -43,6 +45,42 @@ class Timeline : public QObject, public JournallingObject
 		KeepPosition
 	};
 
+	auto pos() const -> const TimePos& { return m_pos; }
+
+	void setPlayPos(TimePos pos)
+	{
+		m_pos = pos;
+		// Calling `setPlayPos` is treated as a jump, so the frame offset is reset
+		m_frameOffset = 0;
+		emit positionChanged();
+		emit positionJumped();
+	}
+
+	auto ticks() const -> tick_t { return m_pos.getTicks(); }
+
+	// By default, calling `setTicks` will emit `positionJumped` signals to update everything accordingly
+	// However, passing `jumped = false` will instead treat the change in ticks as a continuous increment
+	void setTicks(tick_t ticks, bool jumped = true)
+	{
+		if (jumped)
```

Comment:
I think this function is better split in two:
- setTicks(ticks)
- increment/advanceTicks(delta)

since they differ both in second calculation and what signals they emit